tests/unit/nupic/research/temporal_memory_test.py

```diff
@@ -635,8 +1096,9 @@ def testMapCellsToColumns(self):
     self.assertEqual(columnsForCells[99], set([399]))
 
 
-  @unittest.skipUnless(
-      capnp, "pycapnp is not installed, skipping serialization test.")
+  @unittest.skip("Serialization does not preserve numbering of cells\
+                  and segments, whichis needed in the\
+                  columnSegmentWalk implementation")
   def testWriteRead(self):
```

Comment:
I don't think this change should affect whether serialization works. It's true that the C++ implementation makes sure segments and synapses always have the same index before and after a `destroySegment` and `destroySynapse`, but I don't think that matters here.